src/core/Akka.Remote.Tests.Performance/Transports/TestTransportRemoteMessagingThroughputSpec.cs

```diff
@@ -12,22 +12,27 @@ public override Config CreateActorSystemConfig(string actorSystemName, string ip
         {
             var baseConfig = ConfigurationFactory.ParseString(@"
                 akka {
-              actor.provider = ""Akka.Remote.RemoteActorRefProvider,Akka.Remote""
-
-              remote {
-                log-remote-lifecycle-events = off
-
-                enabled-transports = [
-                  ""akka.remote.test"",
-                ]
-
-                test {
-                  transport-class = ""Akka.Remote.Transport.TestTransport,Akka.Remote""
-                  applied-adapters = []
-                  maximum-payload-bytes = 128000b
-                  scheme-identifier = test
+                    loglevel = ""WARNING""
```

Comment:
Added a bunch of logging settings when I was debugging - left them there but turned up the warning level sufficiently high so as to not impact the performance tests themselves.